starter-code/src/functions-and-arrays.js

```diff
@@ -1,4 +1,13 @@
 // Find the maximum
+function maxOfTwoNumbers(num1, num2){
+  if(num1>num2){
+    return num1;
+  }
+    else{
+      return num2;
```

Comment:
Since you have a `return` in the first `if`, if the condition is true, it will stop the function in that moment. So we don't need that else a put only `return num2`